docs/scaling/OVERVIEW.md

```diff
@@ -0,0 +1,41 @@
+ - The readiness and liveness probes defined by the user will be replaced with queue-proxy probes externally to the pod while the user-containers probe endpoints will only be accessed by the proxy.
+ - Makes sure that no more than the ‘defined container concurrency’ requests reach the application's instance at once by queueing other requests locally
+ - Collects metrics about the requests load the application user container and exposes that on a separate port
+
+## Autoscaler
+The Autoscaler is responsible for collecting metrics from the queue-proxy of the different application instances and comes up with the desired the number of pods an application needs using the basic calculation: 
+```
+want = concurrencyInSystem/targetConcurrencyPerInstance
+```
+about the calculations:
+ - Requests stats are stored in a ring buffer indexed by timeToIndex() % len(elements in ring). Each element represents a certain granularity of time, and the total represented duration adds up to a window length of time.
+ - With each new star, a value is added with an associated time to the correct element. If this record would introduce a gap in the data, any intervening times between the last write and this one will be recorded as zero. If an entire window length has expired without data, the firstWrite time is reset, meaning the WindowAverage will be of a partial window until enough data is received to fill it again.
+ - For getting the average over a window of time; the average element value over the window is calculated. If the first write was less than the window length ago, an average is returned over the partial window. For example, if firstWrite was 6 seconds ago, the average will be over these 6 seconds worth of elements, even if the window is 60s. If a window passes with no data being received, the first write time is reset so this behavior takes effect again. 
+ - Similarly, if we have not received recent data, the average is based on a partial window. For example, if the window is 60 seconds but we last received data 10 seconds ago, the window average will be the average over the first 50 seconds. In other cases, for example if there are gaps in the data shorter than the window length, the missing data is assumed to be 0 and the average is over the whole window length inclusive of the missing data.
+ - If too many requests appear in the short time, the Autoscaler panics, which means it decides the required scale based on a shorter window. In normal scenarios, the Autoscaler decides on a trailing average of the past 60 seconds, but in panic mode it's on the last 6 seconds only. This makes the decisions more sensitive to bursty traffic.
+
+In addition to that, the autoscaler adjusts the value for a maximum scale up/down rate and the min- and max-instances settings on the revision. It also computes how much burst capacity is left in the current deployment and thus determines whether or not the activator can be taken off of the data-path or not.
+
+## Activator
```

Comment:
Should we move the activator section before the autoscaler one? The last section of the autoscaler section references bringing the activator on/off the data path, but might be helpful for new users to be introduced to the activator first